storagenode/pieces/trashchore.go

```diff
@@ -80,9 +92,14 @@ func (chore *TrashChore) Run(ctx context.Context) (err error) {
 
 				chore.log.Info("emptying trash started", zap.Stringer("Satellite ID", satellite))
 				trashedBefore := time.Now().Add(-chore.trashExpiryInterval)
-				err := chore.store.EmptyTrash(ctx, satellite, trashedBefore)
+				minPieceTimestamp, err := chore.store.EmptyTrash(ctx, satellite, trashedBefore)
 				if err != nil {
 					chore.log.Error("emptying trash failed", zap.Error(err))
+				} else {
+					chore.log.Info("emptying trash finished", zap.Stringer("Satellite ID", satellite), zap.Time("Next viable run", minPieceTimestamp))
```

Comment:
Log message is incorrect.
```suggestion
					chore.log.Info("emptying trash finished", zap.Stringer("Satellite ID", satellite), zap.Time("Next viable run", minPieceTimestamp.Add(chore.trashExpiryInterval)))
```